aiida/backends/sqlalchemy/migrations/versions/535039300e4a_computer_name_to_label.py

```diff
@@ -19,15 +18,13 @@
 
 def upgrade():
     """Migrations for the upgrade."""
-    op.add_column('db_dbcomputer', sa.Column('label', sa.String(length=255), nullable=False))
-    op.drop_constraint('db_dbcomputer_name_key', 'db_dbcomputer', type_='unique')
+    op.alter_column('db_dbcomputer', 'name', new_column_name='label')  # pylint: disable=no-member
     op.create_unique_constraint('db_dbcomputer_label_key', 'db_dbcomputer', ['label'])
-    op.drop_column('db_dbcomputer', 'name')
+    op.drop_constraint('db_dbcomputer_name_key', 'db_dbcomputer')
```

Comment:
What happens to this constraint when you alter the column? Maybe it's better to drop it before altering (also for performance, so the index does not exist and does not need to be updated while altering)? Same for downgrade (drop before altering)

Reply:
Not sure to be honest. Since this should be a small index (shouldn't be a huge number of computers) I think it won't have a huge impact, but happy to swap the order.

Reply:
swapped them around